environment.yml

```diff
@@ -0,0 +1,42 @@
+name: terramind
+channels:
+  - conda-forge
+  - nodefaults
+  - pytorch
+dependencies:
+  - python=3.13
+  - pip
+  - jupyter
+  - pytorch>2.0
+  - torchvision
+  - rioxarray
+  - rasterio
+  - torchmetrics
+  - geopandas
+  - lightly
+  - jsonargparse<=4.35.0
+  - h5py
+  - hdf5plugin
+  - numba
+  - mlflow
+  - lightning
+  - segmentation-models-pytorch<=0.4
+  - jsonargparse
+  - pytest
+  - torchgeo>=0.7.0
+  - einops
+  - timm>=1.0.15
+  - pycocotools
+  - huggingface_hub
+  - tifffile
+  - python-box
+  - wandb
+  - termcolor
+  - numba
+  - hdf5plugin
+  - h5netcdf
+  - pytest-sugar
+  - pip:
+    - terratorch==1.0.2
```

Comment:
We want to be installing the latest code from their main branch, so this line isn't quite right. Could you switch this to a github link?

Reply:
Yes it's a bit brittle, but not as brittle as relying solely on pip for a complicated dependency tree.